### PR TITLE
add links to forum and reduce opening issues for docs

### PR DESCRIPTION
**Pull request by Borda**
<details>
  <summary><b>Before submitting</b></summary>

- [ ] Was this discussed/agreed via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## What does this PR do?

do not guide users to create issue for typo and add crossroad for forum

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in GitHub issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Ask a Question
+    url: https://lightning.ai/forums
```
Comment (aniketmaurya) [inline .github/ISSUE_TEMPLATE/config.yml:4]: do we use forums? we created [reddit](https://www.reddit.com/r/lightningAI/) recently.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    about: Ask and answer Lightning related questions.
+  - name: 💬 Chat with us
+    url: https://discord.gg/VptPCZkGNa
+    about: Live chat with experts, engineers, and users in our Discord community.
+  - name: 📖 Read the documentation
+    url: https://lightning.ai/lightning-docs/
```
Comment (aniketmaurya) [inline .github/ISSUE_TEMPLATE/config.yml:10]:
```suggestion
    url: https://lightning.ai/litserve/
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    about: Please consult the documentation before opening any issues!
+  - name: 🙋 Contact us about professional services
```
Comment (aniketmaurya) [inline .github/ISSUE_TEMPLATE/config.yml:12]:
```suggestion
  - name: 🙋 Get help with Lightning Studio
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    url: https://lightning.ai
+    about: Contact the Lightning.ai sales team for paid support.
```
Comment (aniketmaurya) [inline .github/ISSUE_TEMPLATE/config.yml:14]:
```suggestion
    about: Contact the Lightning.ai sales team.
```

**Review by aniketmaurya — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Borda) [reply .github/ISSUE_TEMPLATE/config.yml:4]: this may be updated everywhere

**Pull request merged**